javaagent/build.gradle.kts

```diff
@@ -60,7 +51,14 @@ dependencies {
   bootstrapLibs(project(":javaagent-bootstrap"))
 
   // extension-api contains both bootstrap packages and agent packages
-  bootstrapLibs(project(":javaagent-extension-api"))
+  bootstrapLibs(project(":javaagent-extension-api")) {
+    // exclude javaagent dependencies from the bootstrap classpath
+    exclude("net.bytebuddy")
+    exclude("org.ow2.asm")
+    exclude("io.opentelemetry", "opentelemetry-sdk")
+    exclude("io.opentelemetry", "opentelemetry-sdk-extension-autoconfigure")
+    exclude("io.opentelemetry", "opentelemetry-sdk-extension-autoconfigure-spi")
```

Comment:
Just to be 200% sure: have you tried building the agent jar and checking its contents? Are they still as expected?

Reply:
I hadn't, but just checked and looks good 👍